buildSrc/src/main/kotlin/java-library-conventions.gradle.kts

```diff
@@ -145,7 +145,7 @@ val compileModule by tasks.registering(JavaCompile::class) {
 	source = fileTree(moduleSourceDir)
 	destinationDir = moduleOutputDir
 	sourceCompatibility = "9"
-	targetCompatibility = "9"
+	targetCompatibility = "11"
```

Comment:
Because `junit-platform-jfr` targets Java 11 -- will be reverted when we turn it into a MR-JAR with base 8.